packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -266,26 +352,32 @@ const DesignPicker: React.FC< DesignPickerProps > = ( {
 				) }
 			</div>
 
-			<div className="design-picker__grid">
-				{ filteredDesigns.map( ( design, index ) => {
-					return (
-						<DesignCard
-							key={ design.recipe?.slug ?? design.slug ?? index }
-							category={ categorization?.selections.join( ',' ) }
-							design={ design }
-							locale={ locale }
-							isPremiumThemeAvailable={ isPremiumThemeAvailable }
-							shouldLimitGlobalStyles={ shouldLimitGlobalStyles }
-							onChangeVariation={ onChangeVariation }
-							onPreview={ onPreview }
-							getBadge={ getBadge }
-							oldHighResImageLoading={ oldHighResImageLoading }
-							isActive={ showActiveThemeBadge && design.recipe?.stylesheet === siteActiveTheme }
-						/>
-					);
-				} ) }
-				{ filteredDesigns.length === 0 && <NoResults /> }
-			</div>
+			{ isMultiFilterEnabled && (
+				<DesignCardGroup
```

Comment:
do you need to check whether the selected categories are at least 2?

Reply:
Yes please.